.changeset/curly-jars-punch.md

```diff
@@ -0,0 +1,6 @@
+---
+'@emotion/serialize': patch
+'@emotion/utils': patch
+---
+
+Minor performance lints (improved regex, avoid some string work)
```

Comment:
```suggestion
Minor performance improvement
```

Reply:
i like that you mentioned what has been done but this text will get prepended to both CHANGELOGs. Each of those changes is relevant only to one of those packages. It isn't a big deal so I'm just simplifying the message so I can land it right now.

Conceptually there is no 1 changeset 1 PR rule :) we have freedom to have more changesets in a PR, even with overlapping package sets